leader/leader.go

```diff
@@ -44,11 +44,10 @@ var forceRunModeEnv = "OSDK_FORCE_RUN_MODE"
 
```

Comment:
IMO all this stuff will be removed since for the new layout we are using the controller-runtime implementation instead of that.

Reply:
@camilamacedo86 it won't be removed because this is a library for operator developers. This offers leader for life. While it won't be used by the ansible or helm directly, it can be used by other developers.

Reply:
@camilamacedo86 I apologize you are correct the `forceRunModeEnv` and other unused variables can go. They were removed.